tests/integration/test_sns.py

```diff
@@ -196,6 +196,69 @@ def test_unsubscribe_from_non_existing_subscription(
         response = aws_client.sns.unsubscribe(SubscriptionArn=subscription["SubscriptionArn"])
         snapshot.match("empty-unsubscribe", response)
 
+    @pytest.mark.aws_validated
+    @pytest.mark.aws_validated
```

Comment:
nit: we don't need this twice right? :)

Reply:
It's double aws validated!! fixed it, thanks 😄